Api-Login and Profile

Krutikaa04 opened the pull request:
created two APIs -->
POST login api
GET profile api

database collection named users
sample data included

jwt token generated.

Krutikaa04 committed:
api created and added

shahkushal38 commented on app.js:
```diff
@@ -0,0 +1,87 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+require("dotenv").config();
```
This line should come after const jwt line.... ie line 6

shahkushal38 commented on app.js:
```diff
@@ -0,0 +1,87 @@
+  let token;
+
+  if(!username || !password || !email || !branch){
+    return res.status(422).json({error:"fill field properly"});
```
Why response status as 422? Isn't it be 400 

And check for all other cases too please

shahkushal38 commented on app.js:
```diff
@@ -0,0 +1,87 @@
+
+
+    //generate token
+    token = jwt.sign({_id: this._id, email: this.email}, process.env.SECRET_TOKEN, {expiresIn: "30s"});
```
This should not be 30s it should be atleast 24hrs

shahkushal38 commented on model/maindb.js:
```diff
@@ -0,0 +1,63 @@
+
+
+
+// function userInsert(){
```
remove this unwanted code if not needed

shahkushal38 commented on model/schema1.js:
```diff
@@ -0,0 +1,27 @@
+const mongoose = require("mongoose");
```
Please rename the file to userSchema.js to make it understandable

shahkushal38 reviewed (changes requested):
LGTM! Have given suggestions to make the code clean and better to understand

Krutikaa04 committed:
files updated

Krutikaa04 committed:
issues resolved and files updated

Krutikaa04 commented:
Resolved the following : 
1.  status codes changed accordingly
2.  jwt expiry time changed from 30s to 24h
3.  removed unwanted code
4. renamed the file 
5. the database url and the port are now hidden

shahkushal38 reviewed (approved):
LGTM! PR Approved

shahkushal38 reviewed (approved):
approved
